README.md

```diff
@@ -171,10 +171,11 @@ For further details, consult the [wiki](https://github.com/ReactiveX/RxJava/wiki
 
 ## Communication
 
+- [Gitter.im](https://gitter.im/ReactiveX/RxJava)
```

Comment:
Please add this to the bottom without changing the order of the existing list elements.